### DISCUSSION TITLE
New Translation

### DISCUSSION THREAD

**Discussion by sevdimali** [allure-framework/allure2#1896] [Questions & Support] [answered]
Hi, Is it possible to add a new language?
Translate allure to another language?!

**Comment by baev** [comment, marked as answer]
Yes, you can add a translation by submitting a PR to `allure2` repository.  See https://github.com/allure-framework/allure2/pull/1151 as an example. [More examples](https://github.com/allure-framework/allure2/pulls?q=is%3Apr+is%3Amerged+translation+-label%3Atype%3Adependencies+)

**Reply by noomorph** [reply]
Hi, @baev, I have a similar question. Would you accept an [Interslavic (Medžuslovjanky, Меджусловјанскы)](https://en.wikipedia.org/wiki/Interslavic) i18n? 

The work is in progress, but I'd like to know your policy:

![image](https://github.com/allure-framework/allure2/assets/1962469/7abef525-763a-4e46-89e6-9d5be221548b)

**Reply by epszaw** [reply]
I think every locale has a place into the project
